geospace-lib/src/main/scala/com/socrata/geospace/lib/shapefile/MultiLayerShapefileReader.scala

```diff
@@ -0,0 +1,160 @@
+package com.socrata.geospace.lib.shapefile
+
+import java.io.{IOException, File}
+
+import com.socrata.geospace.lib.Utils._
+import com.socrata.geospace.lib.errors.InvalidShapefileSet
+import org.apache.commons.io.FilenameUtils
+import org.geoscript.feature._
+import org.geoscript.layer.Shapefile
+import org.geoscript.projection._
+import org.geoscript.feature.schemaBuilder._
+import org.geoscript.feature._
+import org.geoscript.layer._
+import org.geotools.factory.Hints
+import org.geoscript.projection.Projection
+import org.geotools.referencing.ReferencingFactoryFinder
+import org.opengis.referencing.crs.CoordinateReferenceSystem
+
+/**
+ * Shape file reader but for the case where we have multiple layers.
+ * Notable is that we can also pass a different project if we so desire.
+ * If not, then default is take to be "EPSG:4326"
```

Comment:
Can you add more details about what it means to ingest a multi layer shapefile?   Are the different layers ingested into different 4x4's?  If the same one, how do the different layers integrate?